app/v2/models/request/submitBsas/foreignProperty/ForeignPropertyExpenses.scala

```diff
@@ -43,10 +43,11 @@ case class ForeignPropertyExpenses(premisesRunningCosts: Option[BigDecimal],
     "travelCosts"              -> travelCosts,
     "other"                    -> other,
     "residentialFinancialCost" -> residentialFinancialCost,
-    "consolidatedExpenses"     -> consolidatedExpenses
+    "consolidatedExpense"     -> consolidatedExpenses
   ).collect { case (k, Some(v)) => (k, v) }
 }
 
 object ForeignPropertyExpenses {
-  implicit val format: OFormat[ForeignPropertyExpenses] = Json.format[ForeignPropertyExpenses]
+  implicit val reads: Reads[ForeignPropertyExpenses]   = Json.reads[ForeignPropertyExpenses]
+  implicit val writes: Writes[ForeignPropertyExpenses] = (o: ForeignPropertyExpenses) => Json.toJsObject(o.params)
```

Comment:
The `params` field is only used for validation (and then only the "consolidatedExpenses" and "repairsAndMaintenance" entries). Having to create a separate `validateExpense` method that works around the anomaly in downstream JSON serialization (which is nothing to do with the received request or with its validation) seems wrong.

The intent would be clearer if params were unaltered (or altered to just leave the "consolidatedExpenses" and "repairsAndMaintenance" entries) and this `Writes` was implemented as:
```scala
  // Fix anomalous 'consolidatedExpense' naming for foreign property in downstream spec...
  implicit val writes: Writes[ForeignPropertyExpenses] = (
    (__ \ "premisesRunningCosts").writeNullable[BigDecimal] and
      (__ \ "repairsAndMaintenance").writeNullable[BigDecimal] and
      (__ \ "financialCosts").writeNullable[BigDecimal] and
      (__ \ "professionalFees").writeNullable[BigDecimal] and
      (__ \ "travelCosts").writeNullable[BigDecimal] and
      (__ \ "costOfServices").writeNullable[BigDecimal] and
      (__ \ "residentialFinancialCost").writeNullable[BigDecimal] and
      (__ \ "other").writeNullable[BigDecimal] and
      (__ \ "consolidatedExpense").writeNullable[BigDecimal]
  )(unlift(ForeignPropertyExpenses.unapply))
```
(In an ideal world I would be tempted to fix the `Writes` for other classes too and re-work the validation to remove the need for these `params` maps. But we are where we are... )